orbit-2-test/src/test/java/com/babylon/orbit2/OrbitTestingTest.kt

```diff
@@ -334,7 +334,7 @@ class OrbitTestingTest {
                 sideEffect {
                     post(action)
                 }
-                    .sideEffect { somethingElse(action.toString()) }
+                    .sideEffect { this@SideEffectTestMiddleware.somethingElse(action.toString()) }
```

Comment:
see comment in Host.kt, may be able to revert this part... I think I would find this change frustrating and un-obvious

Reply:
See above, curious what your thoughts are